src/rapids_metadata/json.py

```diff
@@ -41,11 +41,16 @@ def default(
         return dataclasses.asdict(o)
 
 
-def main():
+def main(argv: Union[list[str], None] = None):
+    if argv is None:
+        argv = sys.argv[1:]
+
     parser = argparse.ArgumentParser()
     parser.add_argument("--all-versions", action="store_true")
+    parser.add_argument("--pretty", action="store_true")
+    parser.add_argument("-o", "--output")
```

Comment:
Could you please add `help` entries to these arguments, so that `--help` can be used to understand how the CLI works?

https://docs.python.org/3/library/argparse.html#help

For example, `--output` makes sense to me in the context of this PR, but if I just encountered the CLI somewhere I wouldn't immediately understand from that name that it expects to be passed a filepath.

Reply:
Added help for each argument